framework/branch-key-store.md

```diff
@@ -341,8 +449,10 @@ List of TransactWriteItem:
     - “type“ (S): "branch:version:" + `version`,
     - “enc” (B): the wrapped DECRYPT_ONLY Branch Key `CiphertextBlob` from the KMS operation
     - “create-time” (S): `timestamp`
-    - "kms-arn" (S): configured `KMS Key ARN`
+    - "kms-arn" (S): configured `KMS ARN`
```

Comment:
Hum, nit: A KMS ARN can be an key or an alias. Whereas a KMS Key ARN can only be a key.

Reply:
The problem, it seems, is that we're using `KMS Key ARN` to mean single-Region KMS key, as opposed to `KMS MRK ARN` for multi-Region KMS keys? It's not clear to me but I believe one can use an MRK (and thus its ARN) as if it were a single-Region key, with no notion of regional discovery. So `KMS Key ARN` is accurate in this case; we should keep it here and below to distinguish from aliases.

Reply:
I know. 
But, we called our original KMS Configuration `kmsKeyArn`. 
And then we introduced `kmsMRKeyArn`.

So... if we write "configured `KMS Key ARN`, do we mean never write a `KMS MRKey ARN`?

Which is an ambiguity I am dealing with by saying `KMS ARN`.
Fortunately, we have specified in many other places that it will never be an Alias.

Reply:
I posted my comment without seeing @justplaz comment.
I will refactor.

Reply:
Wait, what if I just put "KMS Key".